packages/ubuntu_init/lib/src/services/privacy_service.dart

```diff
@@ -1,3 +1,6 @@
+// ignore_for_file: avoid_catches_without_on_clauses
```

Comment:
Instead of doing ignore we can disable it in the local analysis file, or are there places where we use catches properly too?

Reply:
Most of the try-catch blocks use a proper `on` clause, so I'd keep the rule active globally and deal with the cases where it's messing.

I'd remove the recently added `on Exception` clauses in the [product service](https://github.com/canonical/ubuntu-desktop-provision/blob/675f89cf5a24c72ba1736d0f989bd7e866b7a9b1/packages/ubuntu_provision/lib/src/services/product_service.dart#L85-L110) and in the refresh service ([here](https://github.com/canonical/ubuntu-desktop-provision/blob/675f89cf5a24c72ba1736d0f989bd7e866b7a9b1/packages/ubuntu_bootstrap/lib/services/refresh_service.dart#L58) and [here](https://github.com/canonical/ubuntu-desktop-provision/blob/675f89cf5a24c72ba1736d0f989bd7e866b7a9b1/packages/ubuntu_bootstrap/lib/services/refresh_service.dart#L72)) and add TODO notes there as well. I don't want to only catch a smaller subset of errors without checking that it doesn't break anything first, but we should definitely clean those up.